Documentation/filesystem.md

```diff
@@ -14,6 +14,11 @@ This example runs a shared file system for the [kube-registry](https://github.co
 
 This guide assumes you have created a Rook cluster as explained in the main [Kubernetes guide](quickstart.md)
 
+### PS
+By default only one shared file system can be created with Rook. Multiple file system support in Ceph is still considered experimental and can be enabled with the environment variable `ROOK_ALLOW_MULTIPLE_FILESYSTEMS`. Enabling this might lead to a scenario where monitors will crash due to an assert failure as described [here](https://github.com/rook/rook/issues/1027)
```

Comment:
A couple suggestions:
- Mention that the environment variable is in the operator.yaml
- Maybe we don't need to include the last sentence since we will pick up the fix from Ceph for that issue hopefully in the next couple of weeks. Or else we just need to remember to remove it after we pick up the fix.

Reply:
If the fix is going to land in a few weeks, i am sure it is alright to not include it now. The fact that it is experimental should be deterrent enough. Should i mention that we recommend that it not be enabled on production clusters?

Reply:
yes, that would be good to mention. Also, what about a more descriptive heading? I haven't seen `PS` used before like this.